src/main/java/seedu/address/logic/commands/DeleteContactedInfoCommand.java

```diff
@@ -39,7 +39,7 @@
             + PREFIX_DELETE_CONTACTED_INFO + "1";
 
     public static final String MESSAGE_DELETE_CONTACTED_INFO_SUCCESS = "Deleted contacted information: %1$s";
-    public static final String MESSAGE_MISSING_INFO = "Index specified out of range.";
+    public static final String MESSAGE_INVALID_CONTACTED_INFO_INDEX = "The specified log entry does not exist.";
     public static final String MESSAGE_EMPTY_LIST = "Cannot delete, list is empty!";
```

Comment:
Maybe a more descriptive variable name and description, like `MESSAGE_EMPTY_CONTACTED_INFO_LIST`?